### PR TITLE
Fix the bug in issue#57

### PR DESCRIPTION
**Pull request by hsj576**
**What type of PR is this?**

/kind bug

**Which issue(s) this PR fixes**:

Fix the bug in issue#57

### CODE REVIEW

File: examples/curb-detection/lifelong_learning_bench/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ cd /ianvs #One might use another path preferred
 
 mkdir project
 cd project
-git clone https://github.com/kubeedge/ianvs.git   
+git clone -b feature-lifelong-n https://github.com/kubeedge/ianvs.git
```
Comment (MooreZheng) [inline examples/curb-detection/lifelong_learning_bench/README.md:29]: feature-lifelong-n  is used in the current version. After merged in the coming version, it should be fine with the previous clone.

Comment (hsj576) [reply examples/curb-detection/lifelong_learning_bench/README.md]: Ok, I agree with that.

File: examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/RFNet/utils/metrics.py [hunk 2]
```diff
@@ -134,7 +134,11 @@ def _generate_matrix(self, gt_image, pre_image):
         return confusion_matrix
 
     def add_batch(self, gt_image, pre_image):
+        gt_image = np.array(gt_image)
+        pre_image = np.array(pre_image)
         print(gt_image.shape, pre_image.shape)
+        if gt_image.shape != pre_image.shape:
+            pre_image = pre_image[0]
```
Comment (luosiqi) [inline examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/RFNet/utils/metrics.py:141]: Changing the dimension of "pre_image" if the condition "!=" is true is not always applicable. Users might have different ways to process pre_image or gt_image before passing them to this function. I suggest that you modify the dimension of "pre_image" outside this function.

**Pull request closed without merging**